lib/conjur/api/router/v4.rb

```diff
@@ -8,18 +8,18 @@ module V4
 
         def authn_login account, username, password
           verify_account(account)
-          RestClient::Resource.new(Conjur.configuration.authn_url, user: username, password: password)['users/login']
+          RestClient::Resource.new(Conjur.configuration.authn_url, Conjur.configuration.rest_client_options.merge(user: username, password: password))['users/login']
```

Comment:
Please break this line up and maybe add a helper variable for `Conjur.configuration.rest_client_options.merge(...)`